build/Dockerfile

```diff
@@ -138,32 +178,40 @@ apt-get install -y --no-install-recommends \
     protobuf-compiler
 EOF
 
-# Build dependencies with cargo chef
-COPY --from=rust-planner ${RUST_BUILD_PATH}/recipe.json .
+# Build external dependencies with cargo chef.
+COPY --from=rust-prepare ${RUST_BUILD_PATH}/recipe.json .
 RUN cargo chef cook --release --recipe-path recipe.json
 
-# Build application
+# Build application.
 COPY ./offchain/ .
-# needed to log commit hash in runtime
-COPY ./.git ../
```

Comment:
I know you did this intentionally, but just for the record: this removes commit info from Rust services' log, but speeds up image-building time by preventing cache invalidation.